### ISSUE TITLE
Argument "text_align" to "table" of "FPDF" has incompatible type "tuple[]"

### ISSUE THREAD

**Issue by matt-b8** [py-pdf/fpdf2#994]
### mypy error
```
error: Argument "text_align" to "table" of "FPDF" has incompatible type "tuple[str, str, str, str]"; expected "str | Align"  [arg-type]
```

### Explanation

The [documentation](https://py-pdf.github.io/fpdf2/Tables.html#setting-text-alignment) shows that you can pass a tuple to ``text_align`` when creating a table:
```python
...
with pdf.table(text_align=("CENTER", "CENTER", "RIGHT", "LEFT")) as table:
    ...
```

But in docstring of ``class Table`` the type of ``text_align`` is declared to ``str, fpdf.enums.Align``
https://github.com/py-pdf/fpdf2/blob/5c078a905cbb08329dbc2cae7f39d68cf840b6be/fpdf/table.py#L120

In ``_render_table_cell`` of ``class Table`` it's actually implemented to process tuple.
https://github.com/py-pdf/fpdf2/blob/5c078a905cbb08329dbc2cae7f39d68cf840b6be/fpdf/table.py#L381-L383

So I think it should be a quick fix of just adding ``tuple`` to supported types? 
```python
"""
text_align (str, fpdf.enums.Align, tuple): ... 
"""
```

**Comment by Lucas-C** [comment]
> So I think it should be a quick fix of just adding tuple to supported types?

Yes!

We have unit tests for both cases in https://github.com/py-pdf/fpdf2/blob/master/test/table/test_table.py

So `tuple[str]` and `tuple[fpdf.enums.Align]` are also valid values.

Would you like to contribute a PR to fix this docstring @matt-b8? 🙂

**Comment by gmischler** [comment]
Fixed in #999.
